cmd/add/schema/schema.go

```diff
@@ -70,48 +55,67 @@ func add(temp int) error {
 		return nil
 	}
 
+	//changing enabled to true based on field number entered
 	conf := config.GetInstance()
-	urlall = conf.AdminConsoleAPIDomain + "/platform-configuration/platform-registration-fields?"
-
-	var resultResp schemaStr
-	resp, err := request.Rest(http.MethodGet, urlall, nil, "")
-
-	if err != nil {
-		return err
-	}
-
-	err = json.Unmarshal(resp, &resultResp)
+	resultResp, err := api.GetFields("all")
 	if err != nil {
-		return err
+		return nil
 	}
 	var temp1 []int
 	for i := 0; i < len(resultResp.Data); i++ {
 		if resultResp.Data[i].Parent == "" {
 			temp1 = append(temp1, i)
 		}
 	}
```

Comment:
What does this means?

Reply:
We are storing the indexes of the standard fields for easier indexing.